test/tests/meshgenerators/transfinite_generator/arccircle_midpoint_generator.i

```diff
@@ -0,0 +1,20 @@
+[Mesh]
+  [transf]
+    type = TransfiniteMeshGenerator
+    corners = '-0.25 0.25 0.0
+              0.25 0.25 0.0
+              0.707106781186547 0.707106781186547 0
+              -0.707106781186547 0.707106781186547 0'
+    nx = 20
+    ny = 6
+    bottom_type = LINE
+    top_type = CIRCARC
+    right_type = LINE
+    left_type = LINE
+    top_parameter = '0.0 && 1.0 && 0.0'
```

Comment:
What about using the `fparse` capability here to illustrate the calculation of the midpoint, like you did for me on slack? Once one knows the pattern, the calculation here is trivial, but it can be a bit obtuse trying to back calculate (especially if one doesn't have 1/sqrt(2) memorized).

Reply:
We actually have a vector of vector input parameter capability that automatically uses `;` as delimiters. `&&` is used nowhere else and would be a bit unexpected/confusing IMO.

Reply:
I agree with @dschwen - let's try to be consistent.

if these are just `x y z`... then they don't need any delimiter (that's how we do it elsewhere)

Reply:
Thanks! I changed to semicolons everywhere.

Reply:
The `&&` separators are still here, aren't they?

Reply:
One case was missed, fixed now. It was under the same edge type and I went by types. Good observation!